vignettes/MetaboCoreUtils.Rmd

```diff
@@ -546,6 +546,100 @@ Generally, injecting study samples in random order can reduce (or even avoid)
 influence of any related technical bias in the downstream analysis and is highly
 suggested to improve and assure data quality.
 
+## Filtering data: Identifying measurement error 
+
+When dealing with metabolomics results, it is often necessary to filter
+features based on certain criteria. These criteria are typically derived
+from statistical formulas applied to full rows of data, where each row
+represents a feature. In this tutorial, we'll explore a set of functions
+designed for filtering metabolomics data.
+
+First, to get more information on the available function you can check the documentation 
+
+```{r}
+?filteringFunctions
+```
+
+We will use a matrix representing metabolomics measurements from different
+samples. Let's start by introducing the data:
+
+```{r}
+# Define sample data for metabolomics analysis
+set.seed(123)
+metabolomics_data <- matrix(rnorm(100), nrow = 10)
+colnames(metabolomics_data) <- paste0("Sample", 1:10)
+rownames(metabolomics_data) <- paste0("Feature", 1:10)
+```
+
+We will begin by calculating the coefficient of variation (CV) for each feature.
+This measure helps assess the relative variability of each metabolite across
+different samples.
+
+```{r}
+# Calculate and display the coefficient of variation
+cv_result <- rowRsd(metabolomics_data)
+print(cv_result)
+```
+
+Next, we will compute the D-ratio, a measure of dispersion, by comparing the
```

Comment:
please add the reference here `[@broadhurst_guidelines_2018]` - either after "D-ratio" or "measure of dispersion"